app/src/main/res/values/strings.xml

```diff
@@ -286,21 +286,21 @@
 
     <string name="pref_header_interface">Interface</string>
 
-    <string name="pref_title_dark_theme">Dark theme</string>
+    <string name="pref_title_dark_theme">Dark</string>
```

Comment:
I do not agree with this change. "Dark" in itself is not enough descriptive for an option under the "Interface" menu, even if it has a sub description.

Reply:
I suppose a middle-ground is preferable.

This is the Android 9 way: 
![image](https://user-images.githubusercontent.com/13802408/47984186-15503000-e0d6-11e8-89a9-3dbb39e1636d.png)
---
![image](https://user-images.githubusercontent.com/13802408/47984207-28fb9680-e0d6-11e8-9050-900bb455671e.png)

It makes some sense in terms of consistency, and I imagine Google to have enough money to run user feedback research.

Reply:
To be honest, this will be redone in a future update. I'd leave it as it is for now.